BothamNetworking/BothamAPIClient.swift

```diff
@@ -134,4 +143,8 @@ public class BothamAPIClient {
         return httpClient.hasValidScheme(request)
 
     }
+}
+
+private struct BothamAPIClientErrorCode {
+    private static let networkError = -1009
```

Comment:
I don't have a status code there :( The only value I have to check is the error code. I'm going to review the errors and update the PR.